### PR TITLE
Feature/delete profile

### PR DESCRIPTION
**Pull request by CloudNiner**

### CODE REVIEW

File: src/nih_wayfinding/app/scripts/views/profile/profiles-controller.js [hunk 1]
```diff
@@ -17,6 +17,32 @@
...
         /**
+         * Open a modal to confirm user deletion, the delete it and unset the currentUser if necessary
+         * @param  {string} name Username of the user to delete
+         */
+        function deleteUser(name) {
+            Modals.openConfirm({
+                text: 'You are about to delete the profile for ' + name + '.',
+                confirmClass: 'btn-danger'
+            }).result.then(function () {
+                ProfileService.deleteProfile(name);
```
Comment (flibbertigibbet) [inline src/nih_wayfinding/app/scripts/views/profile/profiles-controller.js:28]: Does selecting cancel not resolve to this `then` block?

File: src/nih_wayfinding/app/scripts/views/profile/profiles-partial.html [hunk 2]
```diff
@@ -1,18 +1,30 @@
...
         </ul>
     </div>
 </div>
+<footer class="container-fluid">
+    <div class="row">
+        <div class="col-md-6 col-xs-6">
+            <button ui-sref="newprofile" class="btn btn-primary btn-block" type="button">New Profile</button>
+        </div>
+        <div class="col-md-6 col-xs-6">
+            <button ng-click="profiles.deleteMode = !profiles.deleteMode"
```
Comment (flibbertigibbet) [inline src/nih_wayfinding/app/scripts/views/profile/profiles-partial.html:24]: If clicking this button toggles "delete mode" on and off, should probably toggle the button's text, too, maybe to something like "Select Profile" when in delete mode.

**Comment by flibbertigibbet** [comment]
+1

File: src/nih_wayfinding/app/scripts/views/profile/profiles-partial.html (hunk 2 above)
Comment (CloudNiner) [inline src/nih_wayfinding/app/scripts/views/profile/profiles-partial.html:24]: @jfrankl Do you have any opinions on whether this text should change, and if so, what it should change to?

File: src/nih_wayfinding/app/scripts/views/profile/profiles-controller.js (hunk 1 above)
Comment (CloudNiner) [inline src/nih_wayfinding/app/scripts/views/profile/profiles-controller.js:28]: Cancel rejects the promise, so not it does not resolve to this block.

File: src/nih_wayfinding/app/scripts/views/profile/profiles-partial.html (hunk 2 above)
Comment (jfrankl) [inline src/nih_wayfinding/app/scripts/views/profile/profiles-partial.html:24]: Let's not change the text. I think the button changing colors and the trash can icons appearing will be clear. We can test this during user testing.

**Pull request closed without merging**